packages/terra-alert/src/Alert.jsx

```diff
@@ -162,26 +149,20 @@ const Alert = ({
   );
 
   return (
-    <ResponsiveElement
-      tiny={(
-        <div {...attributes} className={narrowAlertClassNames}>
-          <div className={bodyClassNameForNarrowParent}>
+    <ResponsiveElement onChange={value => setBreakpoint(value)}>
+      {
+        breakpoint // Check for a breakpoint before rendering the alert
```

Comment:
This PR may remove the need to perform this check: https://github.com/cerner/terra-core/pull/2761. When 2761 merges can you do some unscripted testing to verify if the flicker is still an issue if the breakpoint check is removed?

Reply:
I tested the changes in #2761 with the changes in this branch, after removing the breakpoint check, I can see flicker happening when render time is greater than 20ms for the component. Below which it's not visible.

Reply:
After implemented the responsive element changes on this branch was the project recompiled? The update to the responsive element should remove the need for the breakpoint check prior to rendering.